recipes/bwidget/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.9.11" %}
+
+package:
+  name: bwidget
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/tcllib/BWidget/{{ version }}/bwidget-{{ version }}.tar.gz
+  sha256: 448a1db08a4ca477ce339dfe592fd85c4c0649c3ff554fb4e86d6826d39bc8b9
+  # use a separate folder to avoid moving scripts generated by conda-build
+  folder: src
+
+build:
+  skip: True  # [win]
+  number: 0
+  script:
+    - cd src
+    # Remove this stuff so that lib is not polluted
+    - rm -Rf .fslckout ChangeLog tests demo BWman
+    - "[[ ! -d $PREFIX/lib/tcl8.6/bwidget ]] || exit 1"
+    - "mkdir -p $PREFIX/lib/tcl8.6/bwidget"
+    - "mv * $PREFIX/lib/tcl8.6/bwidget/"
```

Comment:
This is probably complex enough to be in a `build.sh` script

Reply:
@scopatz moved to `build.sh`

Reply:
moved to `build.sh`